js/logic.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-unused-expressions */
+// eslint-disable-next-line func-names
+const game = (function () {
```

Comment:
You have done well 👍 , You did awesome keeping the DOM from the logic.
Now, let follow the instructions from the Odin;
![image](https://user-images.githubusercontent.com/31897434/76569376-40f5de00-64b3-11ea-90aa-cddee9456359.png)
https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript

In another word; You need to create 3 separate objects, don't forget, no global variable.
1. Gameboard: uses an array to store the 'moves'
2. Player: players will be store as an object, which you already did.
3. Game: control the overflow of the game.


Example:
```
const Player = (x, y) => {
  return { x, y };
};
```
```
const GameBoard = (() => {
  let gameBoard = [ ......... ];
  return { gameBoard };
})();
```

Reply:
thank you for the suggestion we've created a player module to contain all our players